dashboard/db/migrate/20180129221914_add_is_featured_to_featured_projects.rb

```diff
@@ -0,0 +1,5 @@
+class AddIsFeaturedToFeaturedProjects < ActiveRecord::Migration[5.0]
+  def change
+    add_column :featured_projects, :is_featured, :boolean
+  end
+end
```

Comment:
Any reason these are 3 separate migrations?

They go together, and are adding multiple columns to the same table. I'd prefer a single migration.

Reply:
Only reason is that's how I initially learned how to do migrations 😜. Happy to combine into one - what's the convention for naming the migration if it includes multiple changes?

Reply:
Something like `AddTimestampsToFeaturedProjects`.